guides/v2.2/cloud/project/project-start.md

```diff
@@ -7,47 +7,34 @@ functional_areas:
-`/m2-hotfixes` | Used for custom patches.
-`/update` | A service folder used by the support module.
-`.gitignore` | Specify which files and directories to ignore. See [`.gitignore` reference](#ignoring-files).
-`.magento.app.yaml` | Configuration file that defines the properties to build your application. See [Configure environments]({{ page.baseurl }}/cloud/env/environments.html).
-`.magento.env.yaml` | Configuration file that defines actions for the build, deploy, and post-deploy phases. The ece-tools package includes a sample of this file with detailed descriptions for the available variables. See [Configure environments]({{ page.baseurl }}/cloud/env/environments.html).
-`composer.json` | Fetches the Magento Enterprise Edition and the necessary configuration scripts to prepare your application. See [Prepare your Magento install]({{ page.baseurl }}/cloud/setup/first-time-setup-import-prepare.html).
-`composer.lock` | Stores version dependencies for every package.
-`magento-vars.php` | A file used to define [multiple stores]({{ page.baseurl }}/cloud/project/project-multi-sites.html#modify-the-magento-varsphp-file) and sites using [Magento variables]({{ page.baseurl }}/config-guide/multi-site/ms_over.html).
+`/app`                    | The `code` folder is used for custom modules. The `design` folder is used for custom themes. See [Install a theme]({{ page.baseurl }}/cloud/howtos/custom-theme.html). The `etc` folder contains configuration files for Magento.
+`/m2-hotfixes`            | Used for custom patches.
+`/update`                 | A service folder used by the support module.
+`.gitignore`              | Specify which files and directories to ignore. See [`.gitignore` reference](#ignoring-files).
+`.magento.app.yaml`       | Configuration file that defines the properties to build your application. See [Configure environments]({{ page.baseurl }}/cloud/env/environments.html).
+`.magento.env.yaml`       | Configuration file that defines actions for the build, deploy, and post-deploy phases. The ece-tools package includes a sample of this file with detailed descriptions for the available variables. See [Configure environments]({{ page.baseurl }}/cloud/env/environments.html).
+`composer.json`           | Fetches the Magento Enterprise Edition and the necessary configuration scripts to prepare your application. See [Prepare your Magento install]({{ page.baseurl }}/cloud/setup/first-time-setup-import-prepare.html).
+`composer.lock`           | Stores version dependencies for every package.
+`magento-vars.php`        | A file used to define [multiple stores]({{ page.baseurl }}/cloud/project/project-multi-sites.html#modify-the-magento-varsphp-file) and sites using [Magento variables]({{ page.baseurl }}/config-guide/multi-site/ms_over.html).
```

Comment:
These link targets seem confusing --- 

multiple stores ==>  *Modify the `magento-vars.php` file* topic in Cloud docs
Magento variables ==> *Multiple websites or stores*  overview topic in Configuration Guide

Please check

Reply:
This is fine for the table. The Config guide link talks about the Admin UI and defining the Magento variables: `Use the MAGE_RUN_TYPE and MAGE_RUN_CODE variables`
The Cloud link explains the implementation for Cloud and you can see those variables in use.